### ISSUE TITLE
Wrong configuration for Grids on Docs

### ISSUE THREAD

**Issue by bnd170** [Sylius/Sylius#5928]
I'm getting this error creating a new Grid.

> Invalid type for path "sylius_grid.grids.app_grid.driver". Expected array, but got string

I look into Sylius Grids and the config looks diferente as you describe on the docs.
```

sylius_grid:
    grids:
        sylius_admin_administrator:
            driver:
                name: doctrine/orm
                options:
                    class: %sylius.model.admin_user.class%
            sorting:
                createdAt: desc
```

Here is an example as you got on the docs http://docs.sylius.org/en/latest/bundles/SyliusGridBundle/configuration.html#configuration-reference

**Comment by TheMadeleine** [comment]
Solved by #6312. Can be closed @pjedrzejewski, @michalmarcinkowski.

**Comment by michalmarcinkowski** [comment]
@TheMadeleine thanks!
